### PR TITLE
Nav bar2

### PR DESCRIPTION
**Pull request by zoelzer220**
@ryanawalker @kyarber2 

Alright here's my pull request. Couple of things I'm not sure about: 

1. I did remove class="container" out of everyone's file because it was messing up the css. I haven't noticed that this is a bad thing, let me know if you see an issue with it. 
2. I'm having trouble getting the logo to show up on the add/delete ingredient pages. I imagine it's something with those being nested with the mapping. (i.e localhost:8080/ingredients/add
3. When I was testing it and entering a new ingredient -- if I just clicked the button without entering any information, it gave me the error like it should about filling out the name but then the drop-downs were not available to choose. I did not look into this problem. I can if I'm should be the one to do so. 
4. I was also getting an error when adding an ingredient and returning to the ingredient index. I just commented out the 2 lines that were causing the problem (it has to do with the ingredient category and shopping status showing up on the chart). I also did not look into this problem. 
5. I went ahead and messed with the footer as it seemed like it was a similar set up as the nav-bar. I left all Dami's links and thought we could talk about what we wanted there at our next checkin. I did not put it on the ingredients pages, cause I wanted to confirm we wanted it there and I need some help about how to get it on the bottom of the landing page. I can either figure out the correct positioning on the login/register pages or the landing page not both. Possible I should have two different footers? Or there is something I'm missing. 
6. Ryan, I remember you saying about not adding all the files by unchecking the boxes in IntelliJ.. but I'm not sure when I was supposed to do that. So you got all my stuff. I pretty much messed with all the files in resources. As for the seperate .css files I combined Katie's, Dami's and mine all into one. 
7. I also didn't mess with the nav bar saying the username when logged in and the logout button only being there when someone is logged in. Is that something we have enough built out that I could do? 

Hopefully this all makes sense. Sorry for the novel. Hope you guys are having a great holiday.

### CODE REVIEW

File: src/main/resources/templates/ingredients/index.html [hunk 1]
```diff
@@ -19,10 +18,11 @@
     </thead>
     <tr th:each="ingredient : ${ingredients}">
         <td th:text="${ingredient.name}"></td>
-        <td th:text="${ingredient.category.displayName}"></td>
-        <td th:text="${ingredient.toList.displayName}"></td>
+        <td th:text="${ingredient.category}"></td>
```
Comment (kyarber2) [inline src/main/resources/templates/ingredients/index.html]: line 21 needs to have .displayName to show the corresponding enumeration value

File: src/main/resources/templates/ingredients/index.html [hunk 2]
```diff
@@ -19,10 +18,11 @@
...
+        <td th:text="${ingredient.toList}"></td>
```
Comment (kyarber2) [inline src/main/resources/templates/ingredients/index.html]: This also needs .displayName for the enumeration value to appear in the table of ingredients

**Comment by kyarber2** [comment]
@zoelzer220, Looks great!
I also noticed the error regarding the values not showing up once the error message displayed. When I was testing this, I saw that the logo isn't appearing on the add ingredients page or the delete page. I'm not sure if this will be needed or not. Ryan mentioned that the delete page access will likely be limited to administrators. I agree that some of the routing needs to be changed.
Other than that, the footer is floating--I know you mentioned this in our last meeting--and I would also delete any empty files, such as the bootstrap.css file. 
Nice work. The styling is wonderful.

**Comment by zoelzer220** [comment]
@ryanawalker 

Got some updates: 
1. Fixed the footer and the error with the ingredients. Yippee. 
2. Still having trouble getting the logo to show up on the add/delete ingredient pages. I imagine it's something with those being nested with the mapping. (i.e localhost:8080/ingredients/add) any advice for that? 
3. I also didn't mess with the nav bar saying the username when logged in and the logout button only being there when someone is logged in. Is that something we have enough built out that I could do or is that for the story that builds the user home page?

**Review by ryanawalker — APPROVED** [review]
Looks good! See my commits for more info.

**Comment by ryanawalker** [comment]
> 
> 
> @ryanawalker
> 
> Got some updates:
> 
>     1. Fixed the footer and the error with the ingredients. Yippee.
> 
>     2. Still having trouble getting the logo to show up on the add/delete ingredient pages. I imagine it's something with those being nested with the mapping. (i.e localhost:8080/ingredients/add) any advice for that?
> 
>     3. I also didn't mess with the nav bar saying the username when logged in and the logout button only being there when someone is logged in. Is that something we have enough built out that I could do or is that for the story that builds the user home page?

I fixed the logo problem (see commit for more info), but your hunch was right. 👍 Technically we _could_ do the username thing now, but it's probably better to do that in a small bugfix once we have the user page figured out.

**Pull request merged**